src/styles.js

```diff
@@ -1,18 +1,145 @@
 import {createGlobalStyle} from 'styled-components';
 
 export const GlobalStyle = createGlobalStyle`
+
 	*,
 	*::before,
 	*::after {
 		box-sizing: border-box;
 	}
 
 	html {
+		font-family: 'Raleway', sans-serif;
 		font-size: 16px;
 	}
 
 	body {
 		margin: 0;
 		font-size: 1rem;
 	}
+
+	a {
+		color:white;
+		list-style: none;
+		
+	}
+	a:hover {
```

Comment:
pleas refactor and use Styled components :) try not to mix it up. 
If you need to leave it: please think of meaningfull names, no shortcuts ;)